lib/bundler/env.rb

```diff
@@ -86,18 +86,40 @@ def self.git_version
       "not installed"
     end
 
+    def self.version_of(script)
+      return "not installed" unless Bundler.which(script)
+      %x"#{script} --version"
```

Comment:
please don't use straight delimiters, it becomes much harder to read imo. either use normal backticks or something like `%x{}`?